homeassistant/components/logger.py

```diff
@@ -103,6 +109,11 @@ def set_log_levels(logpoints):
             )
         )
 
+    # Set default log severity
+    set_default_log_level('DEBUG')
+    if LOGGER_DEFAULT in config.get(DOMAIN):
```

Comment:
This makes no sense, why call `set_default_log_level` twice, use if…else

Reply:
I agree, but I wanted to keep it to as close as it was as possible, and the original set it and then set it again. I'll update it.